src/layout/GeometryMap/GeometryMapComponent.tsx

```diff
@@ -0,0 +1,193 @@
+import type { GeometryMapLocation } from 'src/layout/GeometryMap/config.generated';
+
+export type IGeometryMapComponentProps = PropsFromGenericComponent<'GeometryMap'>;
+
+export function GeometryMapComponent({ isValid, node }: IGeometryMapComponentProps) {
+  const { readOnly, centerLocation, zoom, dataModelBindings } = node.item;
+
+  const layers: MapLayer[] = [
+    {
+      url: 'https://opencache.statkart.no/gatekeeper/gk/gk.open_gmaps?layers=europa_forenklet&zoom={z}&x={x}&y={y}',
+      attribution: 'Data © <a href="https://www.kartverket.no/">Kartverket</a>',
+    },
+    {
+      url: 'https://opencache.statkart.no/gatekeeper/gk/gk.open_gmaps?layers=norgeskart_bakgrunn2&zoom={z}&x={x}&y={y}',
+      attribution: 'Data © <a href="https://www.kartverket.no/">Kartverket</a>',
+    },
+  ];
+
+  const coordinates: any = [];
+  const labels: any = [];
+
+  const formData = FD.useFreshBindings(dataModelBindings, 'raw');
+  if (Array.isArray(formData['array'])) {
+    for (const obj of formData['array']) {
+      const arrayBinding = dataModelBindings['array'];
+      const textBinding = dataModelBindings.label.replace(`${arrayBinding}.`, '');
+      const wktStringBinding = dataModelBindings.wkt.replace(`${arrayBinding}.`, '');
+      coordinates.push(dot.pick(wktStringBinding, obj));
+      labels.push(dot.pick(textBinding, obj));
+    }
+  } else {
+    coordinates.push(formData['wkt']);
+    labels.push(formData['label']);
+  }
+
+  const [inputCoords, geometryType] = findCoordinates(coordinates);
+
+  const polyCenter = findPolygonCenter(inputCoords[0][0]);
+
+  const DefaultCenterLocation: Location = {
+    latitude: 64.888996,
+    longitude: 12.8186054,
+  };
+
+  const center = polyCenter
+    ? polyCenter
+    : centerLocation
+      ? locationToTuple(centerLocation)
+      : locationToTuple(DefaultCenterLocation);
+
+  const [map, setMap] = useState<LeafletMap | null>(null);
+
+  return (
+    <div className={`geometry-map-component${isValid ? '' : ' validation-error'}`}>
+      <MapContainer
+        className={classes.map}
+        center={center}
+        ref={setMap}
+        zoom={zoom ? zoom : polyCenter ? 4 : 16}
+        dragging={!readOnly}
+        attributionControl={false}
+      >
+        {layers.map((layer, i) => (
+          <TileLayer
+            key={i}
+            url={layer.url}
+            attribution={layer.attribution}
+            subdomains={layer.subdomains ? layer.subdomains : []}
+            opacity={readOnly ? 0.5 : 1.0}
+          />
+        ))}
+        <AttributionControl prefix={false} />
+
+        {inputCoords.map(
+          (coords, i) =>
+            geometryType[i] === 'polygon' ?? (
+              <Polygon
+                key={i}
+                positions={coords}
+              >
+                <Tooltip>{labels !== undefined ? <span>{JSON.stringify(labels[i])}</span> : <div></div>}</Tooltip>
+              </Polygon>
+            ),
+        )}
```

Comment:
Skjønner ikke helt bruken av nullish coalescing (`??`) her. Vil ikke dette bare returnere en `<Polygon>` dersom `geometryType[i] === 'polygon'` er `null`? Noe som aldri bør skje? Jeg antar at du har testa at det fungerer, men hva er tanken her?

Reply:
Takk for at du fant dette! Testet om det ville gå tidligere så man kunne slippe det som nå er etter kolon der, det hadde visst sneket seg med på en push som gjorde noe annet dog, men er fikset nå!
Om det er bedre å ta null istedenfor en tom div slik det er nå må du bare skrike ut så fikser jeg det!